.travis.yml

```diff
@@ -2,8 +2,8 @@ sudo: false
 language: go
 
 go:
- - 1.5.4
- - 1.6.2
+ - 1.6
```

Comment:
It's more a problem with how Go treats versioning.
They call 1.6.0 in fact just 1.6, so I guess we have to explicitly specify the latest bugfix version.